.github/workflows/test.yml

```
@@ -1,33 +1,90 @@
-# This is a basic workflow to help you get started with Actions
-
 name: CI
-
-# Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the main branch
   push:
     branches:
       - main
   pull_request:
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
-
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  test-flow:
+  jest:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [6, 7, 8, 9, 10, 12]
```

Node 10 is the oldest currently supported node release 
```suggestion
        node: [10, 12, 14, 15]
```